libqtile/manager.py

```diff
@@ -312,6 +315,18 @@ def gobject_thread():
             logger.warning("importing dbus/gobject failed, dbus will not work.")
             self._glib_loop = None
 
+    def configure_dpi(self):
+        # For whatever reason, self.conn.default_screen.width_in_millimeters is
+        # basically just wrong most of the time. Instead, we query xrandr and
+        # add up each of the outputs individually.
+        dpi = self.conn.randr.find_dpi(self.root.wid)
+        dpi = dpi * self.config.dpi_scale
+
+        self.root.set_property("RESOURCE_MANAGER", "Xft.dpi: %d\n" % dpi,
```

Comment:
There could be a _slight_ difference between the xft dpi (int) and the pango dpi (float), will this even be noticeable?

Reply:
I don't think so, unless you have really good eyes. At the end of the day it's all pixel level interfaces anyway, so it really is ints :)

Reply:
Your comment about width in millimeters reminded me of this: https://download.nvidia.com/XFree86/Linux-x86_64/304.137/README/dpi.html